### PR TITLE
Remove bad reference from atmos

### PR DESCRIPTION
**Pull request by Ghilker**
## About The Pull Request
Change the name from cyrion B to Zauker due to the easy connection the bad groups in the past

edit: the name is taken from an old italian comic from a satirical journal (https://en.wikipedia.org/wiki/Don_Zauker)

## Why It's Good For The Game
Not getting a repo ban is good

## Changelog
:cl:
tweak: Cyrion-B is now named Zauker
/:cl:

### CODE REVIEW

**Comment by Ghilker** [comment]
Thanks no thanks to @EdgeLordExe for the reference :angery:

**Comment by Barhandar** [comment]
How is Zauker any less of a reference?

**Comment by Ghilker** [comment]
whats wrong with zauker?

**Review by Rohesie — DISMISSED** [review]

File: code/modules/power/supermatter/supermatter.dm [hunk 1]
```diff
@@ -37,7 +37,7 @@
 #define HYDROGEN_TRANSMIT_MODIFIER 25 //increase the radiation emission, but less than the trit (2.5)
 #define HEALIUM_TRANSMIT_MODIFIER 2.4
 #define PROTO_NITRATE_TRANSMIT_MODIFIER 15
-#define CYRION_B_TRANSMIT_MODIFIER 20
+#define taburine_TRANSMIT_MODIFIER 20
```
Comment (bobbahbrown) [inline code/modules/power/supermatter/supermatter.dm:40]: please review the changes made by your find-and-replace before making a PR, constants should have uppercased names.

Comment (Ghilker) [reply code/modules/power/supermatter/supermatter.dm:40]: Ah thanks I'll change it

**Review by Mothblocks — DISMISSED** [review]

**Comment by Jared-Fogle** [comment]
Out of curiosity, how do you even pronounce this?

**Comment by Ghilker** [comment]
is similar to taurine, just add a b

**Review by Twaticus — APPROVED** [review]
its not even close but ok lol

**Comment by msgerbs** [comment]
Can somebody help a naive fool? What is the current name a reference to? I only found Warhammer 40K stuff?

**Comment by Jared-Fogle** [comment]
@msgerbs My money was on Zyklon B.

**Comment by msgerbs** [comment]
Oh shit. I did not see that reference. Yeah, probably best not to have that in. I'd bet good money that's where Warhammer got it from.On Sep 17, 2020 1:36 AM, Jared-Fogle <notifications@github.com> wrote:
@msgerbs My money was on Zyklon B.

**Comment by Barhandar** [comment]
Is Zyklon B specifically the horrible terrible no good very bad must be deleted reference, or "real toxic gas" in general? Zauker(berg) was better than the Tabun-ine in that regard.

Also there's a perfect SS13 poison, [delta atracotoxin](https://en.wikipedia.org/wiki/Robustoxin), also known as "robustoxin". Not a gas, but that doesn't really matter.

**Comment by Ghilker** [comment]
You have good points (Cyrion B and Zauker were indeed better names)
Just to point out tho Zauker is not a reference to Zuckerberg, but is a character in an Italian humoristic journal, you can read more here https://en.wikipedia.org/wiki/Don_Zauker if people judge that this name does not have offensive connections, i'll change Taburine gas (sucks) to Zauker gas (much better and scary)

**Comment by Barhandar** [comment]
I know it has no reference to Zuckerberg inherently, I am making a joke (and others will likely do so too).

Fun fact: [the original name is borderine Don Sauce.](https://en.wikipedia.org/wiki/Daitarn_3#Main_characters)

**Comment by Ghilker** [comment]
Fun fun fact: the name is taken from exactly that anime/manga

**Comment by LemonInTheDark** [comment]
Jesus christ how did I miss this.

**Pull request merged**